test/Project.toml

```diff
@@ -16,6 +16,7 @@ Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
+T8code = "d0cc0030-9a40-4274-8435-baadcfd54fa1"
```

Comment:
TODO: Remove T8code when possible

Reply:
Done in [819014f](https://github.com/trixi-framework/Trixi.jl/pull/1844/commits/819014f3a609635d5f86d94ae0f2031fe07b5158)